lib/client-api/src/types.ts

```diff
@@ -39,6 +39,7 @@ export interface StoreSelectionSpecifier {
   viewMode: ViewMode;
   singleStory?: boolean;
   args?: Args;
+  globalArgs?: Args;
```

Comment:
We renamed `globalArgs` to `globals` everywhere; I suggest we don't add it back now. cc @tmeasday

Reply:
Wasn't aware of that. Renamed.